internal/cmd/argus/instance/list/list.go

```diff
@@ -0,0 +1,142 @@
+)
+
+const (
+	limitFlag = "limit"
+)
+
+type inputModel struct {
+	*globalflags.GlobalFlagModel
+	Limit *int64
+}
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "Lists all Argus instances",
+		Long:  "Lists all Argus instances.",
+		Args:  args.NoArgs,
+		Example: examples.Build(
+			examples.NewExample(
+				`List all Argus instances`,
+				"$ stackit argus instance list"),
+			examples.NewExample(
+				`List all Argus instances in JSON format`,
+				"$ stackit argus instance list --output-format json"),
+			examples.NewExample(
+				`List up to 10 Argus instances`,
+				"$ stackit argus instance list --limit 10"),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			ctx := context.Background()
+			model, err := parseInput(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Configure API client
+			apiClient, err := client.ConfigureClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			// Call API
+			req := buildRequest(ctx, model, apiClient)
+			resp, err := req.Execute()
+			if err != nil {
+				return fmt.Errorf("get Argus instances: %w", err)
+			}
+			instances := *resp.Instances
+			if len(instances) == 0 {
+				projectLabel, err := projectname.GetProjectName(ctx, cmd)
+				if err != nil {
+					projectLabel = model.ProjectId
+				}
+				cmd.Printf("No instances found for project %q\n", projectLabel)
+				return nil
+			}
+
+			// Truncate output
+			if model.Limit != nil && len(instances) > int(*model.Limit) {
+				instances = instances[:*model.Limit]
+			}
+
+			return outputResult(cmd, model.OutputFormat, instances)
+		},
+	}
+
+	configureFlags(cmd)
+	return cmd
+}
+
+func configureFlags(cmd *cobra.Command) {
+	cmd.Flags().Int64(limitFlag, 0, "Maximum number of entries to list")
+}
+
+func parseInput(cmd *cobra.Command) (*inputModel, error) {
+	globalFlags := globalflags.Parse(cmd)
+	if globalFlags.ProjectId == "" {
+		return nil, &errors.ProjectIdError{}
+	}
+
+	limit := flags.FlagToInt64Pointer(cmd, limitFlag)
+	if limit != nil && *limit < 1 {
+		return nil, &errors.FlagValidationError{
+			Flag:    limitFlag,
+			Details: "must be greater than 0",
+		}
+	}
+
+	return &inputModel{
+		GlobalFlagModel: globalFlags,
+		Limit:           flags.FlagToInt64Pointer(cmd, limitFlag),
+	}, nil
+}
+
+func buildRequest(ctx context.Context, model *inputModel, apiClient *argus.APIClient) argus.ApiListInstancesRequest {
+	req := apiClient.ListInstances(ctx, model.ProjectId)
+	return req
+}
+
+func outputResult(cmd *cobra.Command, outputFormat string, instances []argus.ProjectInstanceFull) error {
+	switch outputFormat {
+	case globalflags.JSONOutputFormat:
+		details, err := json.MarshalIndent(instances, "", "  ")
+		if err != nil {
+			return fmt.Errorf("marshal Argus instance list: %w", err)
+		}
+		cmd.Println(string(details))
+
+		return nil
+	default:
+		table := tables.NewTable()
+		table.SetHeader("ID", "PLAN", "NAME", "STATUS")
```

Comment:
Maybe swap the order of plan with name? Or did you take this from some other instance that has it like this?